Source/WebCore/rendering/TextDecorationPainter.cpp

```diff
@@ -141,7 +141,7 @@ static DashArray translateIntersectionPointsToSkipInkBoundaries(const DashArray&
                 intermediateTuples.append(*i);
         }
     } else
-        intermediateTuples = tuples;
+        intermediateTuples.swap(tuples);
```

Comment:
I would add a one-line comment to explain that this is due to a suspected compiler bug. Otherwise, somebody is going to become confused why there is a swap here when `tuples` is not used after this point, and will "simplify" this back to an assignment in the future.

Reply:
Done, commment added before landing 👍🏼